wum-live-sync-automation/uatBatchJob/Jenkinsfile

```diff
@@ -0,0 +1,176 @@
+import groovy.sql.Sql
+import java.io.File
+import java.nio.file.*
+import groovy.json.*
+
+def boolean runShellCommands(env, wumProdBaseLoc, cmd, wumUpdateFileNameList, wsUpdateFolderName) {
```

Comment:
It would be good to add a comment mentioning the purpose of this build job and this method.

Reply:
Added doc comments